.travis.yml

```diff
@@ -25,27 +25,27 @@ matrix:
     - python: '3.7'
       env:
         - TOXENV="py37-linux"
-        - BOTO_CONFIG="/dev/null"
+        - BOTO_CONFIG="/dev/null"  # XXX: why is this here?
```

Comment:
@menshikh-iv can you comment here as well?

Reply:
I think this is one of my creations, this ensures that boto3 does _not_ read any config files lying around in `~/.aws/`.

Reply:
Related to https://github.com/travis-ci/travis-ci/issues/7940, can be removed now I guess